packages/tools/tsconfig/ts-library.json

```diff
@@ -3,6 +3,9 @@
   "display": "Typescript Library",
   "extends": "./base.json",
   "compilerOptions": {
+    "composite": true,
```

Comment:
interesting. https://www.typescriptlang.org/tsconfig#composite

imo may be worth documenting inline why this is necessary otherwise it may be lost to the sands of time

Reply:
Done